File: 50 Coding Challenge Part II/JavaScript/Challenge37.js
// Challenge 37: Create a function that will convert a string in an array containing the ASCII
// codes of each character

function getCharCode(string){
    let arr = [];

    for (let i = 0; i < string.length; i++){
        let code = string.charCodeAt(i);
        arr.push(code);
    }
    return arr;
}


// Driver Method
console.log(getCharCode("I love JavaScript"));

File: 50 Coding Challenge Part II/Python/Challenge37.py
# Challenge 37: Create a function that will
# convert a string in an array containing the ASCII
# codes of each character

def getCharCode(text):
    arr = []
    for char in text:
        code = ord(char)
        arr.append(code)

    return arr


# Driver Method
print(getCharCode("I love JavaScript"))
